scripts/docker-integration-tests/carbon/expected/a.b.c.d.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "id": "a.b*.caw.daz",
```

Comment:
by the looks of it there are no `*` in the ids when they're ingested, why is there still a `*` in the id returned here?

Reply:
It's a quirk of graphite where it includes your query string in the `id`; just tried querying statsdex for `foo.bar.que*.*` [actual metrics scrubbed] and got the following:

`[ {"id":"foo.bar.que*.baz","text":"baz","leaf":0,"expandable":1,"allowChildren":1}, ... ]`

Reply:
Yech, aight.